src/Dashboard/styles.css

```diff
@@ -1,4 +1,17 @@
 .dashboard {
     display: flex;
     flex-flow: column;
+    width: 700px;
+    margin: auto;
+}
+
+.dashboardTodoSearch {
+    display: flex;
+    align-self: center;
+    margin-left: 8%;
```

Comment:
Percent margin never works well on dynamic layouts. Use flex-style properties to set position of element